webapp/routes.go

```diff
@@ -13,24 +13,27 @@ import (
 var templates = template.Must(template.ParseGlob("templates/*.html"))
 
 func init() {
-	// Test run results, viewed by browser (default view)
-	// For run results diff view, 'before' and 'after' params can be given.
-	shared.AddRoute("/", testResultsHandler)
-	shared.AddRoute("/results", testResultsHandler) // Prevent default redirect
-	shared.AddRoute("/results/", testResultsHandler)
-
 	// About wpt.fyi
-	shared.AddRoute("/about", aboutHandler)
-
-	// Test run results, viewed by pass-rate across the browsers
-	shared.AddRoute("/interop/", interopHandler)
+	shared.AddRoute("/about", "about", aboutHandler)
 
 	// Lists of test run results which have poor interoperability
-	shared.AddRoute("/interop/anomalies", anomalyHandler)
+	shared.AddRoute("/anomalies", "anomaly", anomalyHandler)
+
+	// Test run results, viewed by pass-rate across the browsers
+	shared.AddRoute("/interop/", "interop", interopHandler)
```

Comment:
Why is this the only route that has a trailing slash?

Reply:
A trailing slash and `StrictSlash(true)` mean that the slash is optional, and a redirect is added from the non-slash to the slash.

The test `path` part of the url after `/results` or `/interop` is expected in the UI, so a trailing slash is used in these paths.